lucene/spatial-extras/src/java/org/apache/lucene/spatial/spatial4j/Geo3dRectangleShape.java

```diff
@@ -30,32 +30,49 @@
 import org.locationtech.spatial4j.shape.SpatialRelation;
 
 /**
- * Specialization of a {@link Geo3dAreaShape} which represents a {@link Rectangle}. This
+ * Specialization of a {@link Geo3dShape} which represents a {@link Rectangle}. This
  * class is used for geohashing.
```

Comment:
minor: not need to say "this class is used for geohashing".  It could be used for a variety of purposes, notably a query for a rectangular map area.